include/deal.II/distributed/tria_base.h

```diff
@@ -704,12 +703,12 @@ namespace parallel
      * description of the latter, see the documentation for the member
      * function register_data_attach().
      */
-    using cell_relation_t = typename std::tuple<CellStatus, cell_iterator>;
+    using cell_relation_t = typename std::pair<cell_iterator, CellStatus>;
 
     /**
-     * Vector of tuples, which each contain a deal.II cell
+     * Vector of pair, each containing a deal.II cell iterator
      * and their relation after refinement. To update its contents, use the
-     * compute_cell_relations member function.
+     * update_cell_relations() member function.
      *
      * The size of this vector is assumed to be equal to the number of locally
      * owned quadrants in the parallel_forest object.
```

Comment:
Their relation to what exactly? After removing the p4est quadrant, we need to be more specific.

The deal.II cell belongs to the old mesh before refinement, the CellStatus or relation describes how this old cell will be changed on the new mesh. You might refer to the documentation of the `register_data_attach` function for more information on CellStatus.

Also, we don't have a parallel_forest in TriaBase anymore, correct? I think the size of the vector then corresponds to the number of active cells on the new mesh? Maybe you can elaborate in which order the cells will be inserted.

For distributed Triangulation, each entrie's index corresponds to the position of the p4est quadrant in the underlying p4est sc_array, i.e., `tree.quadrants_offset + idx` where idx is the position of the quadrant in its corresponding tree. Do you have a specific order in which you insert the information in fullydistributed tria?

Reply:
Thanks! I have updated the documentation. The order the information is stored is documented accordingly in each derived class `p::d::T`/`p::f::T`.